librz/core/cconfig.c

```diff
@@ -1130,6 +1130,16 @@ static bool cb_str_search_max_threads(void *user, void *data) {
 	return true;
 }
 
+static bool cb_search_max_threads(void *user, void *data) {
+	RzConfigNode *node = (RzConfigNode *)data;
+	RzThreadNCores max_threads = rz_th_max_threads(node->i_value);
+	if (node->value[0] == '?') {
+		rz_cons_printf("%d\n", max_threads);
```

Comment:
This should return ncores. It's under rz_th.h

Reply:
`max_threads = rz_th_physical_core_number();`

Reply:
This is not resolved.

Reply:
Sorry, I guess I didn't get it then.
The idea is to set the number of threads. And it prints the maximum number of cores, if the user types `'?'`.

Reply:
The function gets called before the check on the question mark